internal/controller/pdbwatcher_controller.go

```diff
@@ -162,7 +187,17 @@ func (r *PDBWatcherReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 	}
 
-	// What will return us to the original state? Assuming we need a watch on PDB for it to come off
+	// Watch for changes in PDB to revert to original state
+	if pdb.Status.DisruptionsAllowed > 0 && *deployment.Spec.Replicas != pdbWatcher.Status.MinReplicas {
```

Comment:
using pdbWatcher.Status.MinReplicas  here is a little suspect because its going to be updaed next time we reconcile by pdbWatcher.Status.MinReplicas = *deployment.Spec.Replicas

So not sure this is a good way to add this. Might want to instead track the number of extra replicas our watcher has added.